src/hotspot/share/jvmci/jvmciRuntime.hpp

```diff
@@ -35,6 +35,8 @@
 #include "gc/g1/g1CardTable.hpp"
 #endif // INCLUDE_G1GC
 
+#define JVMCI_NOT_ENABLED_ERROR_MESSAGE "JVMCI is not enabled. Must specify '--add-modules=jdk.internal.vm.ci' or '-XX:+EnableJVMCI' to the java launcher."
```

Comment:
It's the exception message for an InternalError so maybe this is okay but in general I don't think you want to be asking users to have to name a JDK internal module.

Reply:
Assuming we want users to use `-XX:+EnableJVMCI` over `--add-modules=jdk.internal.vm.ci` for said reason, maybe flip the order?

Reply:
Done/